lib/tizen_doctor.dart

```diff
@@ -89,18 +89,27 @@ class TizenValidator extends DoctorValidator {
       return ValidationResult(ValidationType.missing, messages);
     }
 
-    final double sdkVersion = double.tryParse(tizenSdk.sdkVersion) ?? 0;
-    if (sdkVersion < 4.0) {
+    if (tizenSdk.sdkVersion == null) {
```

Comment:
If `tizenSdk.sdkVersion` cannot be parsed, it is better to handle it here. 
```suggestion
    final double sdkVersion = double.tryParse(tizenSdk.sdkVersion ?? '');
    if (sdkVersion == null) {
```

Reply:
I updated the code.

I personally don't like the hack that makes use of `''`, but this way looks functionally correct.